builder/builder.mk

```diff
@@ -24,6 +24,19 @@ BUILDER = BUILDER_NAME=$(BUILDER_NAME) $(abspath $(BUILDER_HOME)/builder.sh)
 DBUILD = $(abspath $(BUILDER_HOME)/dbuild.sh)
 COPY_GOLD = $(abspath $(BUILDER_HOME)/copy-gold.sh)
 
+# the image used for running the Ingress v1 tests with KIND
+# build this image with:
+# 1. checkout the Kuberentes sources in a directory like "~/sources/kubernetes"
+# 2. kind build node-image --kube-root ~/sources/kubernetes
+# 3. docker tag kindest/node:latest quay.io/datawire/kindest-node:latest
+# 4. docker push quay.io/datawire/kindest-node:latest
+# This will not be necessary once the KIND images are built for a Kuberentes that support Ingress v1
```

Comment:
Just being picky, but the comment should say "... once the KIND images are built for a Kubernetes 1.18 and support Ingress v1beta1 improvements"